scripts/logs-backup-from-all-pods/get-all-pod-logs.sh

```diff
@@ -2,6 +2,19 @@
 # and tar and gzip those files
 #!/bin/bash
 ns=$1
+ISD_ADMIN_USERNAME=$2
+ISD_ADMIN_PASSWORD=$3
+REDIS_HOST=$4
+REDIS_PORT=$5
+REDIS_PASSWORD=$6
+ISD_GATE_URL=$7
+DAYS=$8
+INSTALLATION_TYPE=$9
+PLATFORM_DB=${10}
+PLATFORM_HOST=${11}
+USER_NAME=${12}
+PASSWORD=${13}
+PORT=${14}
 [ -z "$ns" ] && echo "Please pass namespace as an argument"
 [ -z "$ns" ] && exit 1
 FILE=logs.tar            
```

Comment:
Where are we planning to execute this shell script ? is it inside platform pod ? if not then this script will fail as only platform pod has all the python libraries installed

Reply:
We have a job that creates the necessary environment to execute this script.